### ISSUE TITLE
Line number in raport is wrong

### ISSUE THREAD

**Issue by richardwerkman** [stryker-mutator/stryker-net#174]
The line numbers of mutations are displayed incorrect. They seem to be always +1. This can be confusing for users as they cannot find the right line when looking for the mutation in their code.

```
[Survived] Binary expression mutation on line 39: 'request.Limit.Value <= 0' ==> 'request.Limit.Value < 0'
```
Can be found on line 40 in the source code. I'm not sure where the wrong number comes from.

**Comment by nicojs** [comment]
There are 3 hard problems in software development: cache invalidation and off-by-one errors. :trollface:

**Comment by Szpi** [comment]
I can take this issue.

**Comment by Szpi** [comment]
PR #176

**Comment by simondel** [comment]
Closed with #176
